src/frontend.md

```diff
@@ -5,31 +5,57 @@
 ```
 src/
   components/ (can be dumb and smart)
-    bankAccount/
+    [All components should be placed in some entity folder or shared folder]
+    bankAccount/ (entity, module, .etc)
       BankAccountTable/
         [no nested directories]
+        [avoid index.ts]
+        [other files should be placed in queries or services]
         BankAccountTable.ts
-        BankAccountTable.test.ts
+        BankAccountTable.test.ts (unit tests)
+        BankAccountTable.spec.ts (integration tests)
         BankAccountTableBadge.ts (child component)
+        useBankAccountTableBadge.ts (ui hook used only in the current component)
         BankAccountTable.styled.ts
+        useBankAccountTable.ts (ui hook used only in the current component)
```

Comment:
I guess we should briefly define what do ui/non-ui hooks mean
since it depends on where we should place hooks (components or services)